index.html

```diff
@@ -30,30 +30,30 @@
   <body>
     <div class="container">
       <header class="app-bar">
-        <div class="logo purple">
+        <div class="logo bold dark-purple">

```

Comment:
- [optional] you can make the logo and hamburger static, so they don't overlap, or you give them their background color to signify that it is the header.